drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -95,6 +98,50 @@ def ssn_serial
     self.SSN&.[](-4..-1)
   end
 
+  def warehouse_id
+    destination_client.id
```

Comment:
Maybe this would be better as `warehouse_client_source.destination_id` to keep the interaction with the warehouse data structures to a minimum? (also one fewer query/join)